### PR TITLE
Gateway

### PR DESCRIPTION
**Pull request by OskarSundberg**
Not done yet!
But so you can see what i am doing :+1:

### CODE REVIEW

File: gateway/dist/index.js
Comment (OskarSundberg) [inline gateway/dist/index.js:1]: denna borde inte vara här.... D:

Comment (Linkan42) [reply gateway/dist/index.js:1]: huh, why not

**Comment by Linkan42** [comment]
lookin sharp

**Review by Linkan42 — DISMISSED** [review]

File: gateway/dist/index.js
Comment (OskarSundberg) [reply gateway/dist/index.js:1]: done now

**Review by ivonilsson — APPROVED** [review]
https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExeTU4eG13ejUyNnhhNnE2N2hmbjFiMXZxandweGFxajkxejI2ejB4bSZlcD12MV9naWZzX3NlYXJjaCZjdD1n/jihwEDnsFoaXWDTiKc/giphy.gif

**Pull request merged**